src/cli/init.rs

```diff
@@ -579,4 +605,56 @@ mod tests {
 
         dir.close().unwrap();
     }
+
+    #[test]
+    fn test_multiple_format_values() {
+        let test_cases = vec![
+            ("pixi", ManifestFormat::Pixi),
+            ("PiXi", ManifestFormat::Pixi),
+            ("PIXI", ManifestFormat::Pixi),
+            ("pyproject", ManifestFormat::Pyproject),
+            ("PyPrOjEcT", ManifestFormat::Pyproject),
+            ("PYPROJECT", ManifestFormat::Pyproject),
+        ];
+
+        for (input, expected) in test_cases {
+            let args = Args::try_parse_from(&["init", "--format", input]).unwrap();
+            assert_eq!(args.format, Some(expected));
+        }
+    }
+
+    #[test]
+    fn test_multiple_scm_values() {
+        let test_cases = vec![
+            ("github", GitAttributes::Github),
+            ("GiThUb", GitAttributes::Github),
+            ("GITHUB", GitAttributes::Github),
+            ("Github", GitAttributes::Github),
+            ("gitlab", GitAttributes::Gitlab),
+            ("GiTlAb", GitAttributes::Gitlab),
+            ("GITLAB", GitAttributes::Gitlab),
+            ("codeberg", GitAttributes::Codeberg),
+            ("CoDeBeRg", GitAttributes::Codeberg),
+            ("CODEBERG", GitAttributes::Codeberg),
+        ];
+
+        for (input, expected) in test_cases {
+            let args = Args::try_parse_from(&["init", "--scm", input]).unwrap();
+            assert_eq!(args.scm, Some(expected));
+        }
+    }
+
+    #[test]
+    fn test_invalid_scm_values() {
```

Comment:
Not sure how much value this test add but it doesn't hurt to have an extra test case.